kong/core/balancer.lua

```diff
@@ -0,0 +1,79 @@
+local dns_client = require "dns.client"
+
+local toip = dns_client.toip
+
+-- looks up a balancer for the target.
+-- @param target the table with the target details
+-- @return balancer if found, or nil if not found, or nil+error on error
+local get_balancer = function(target)
+  return nil  -- TODO: place holder, forces dns use to first fix regression
```

Comment:
@Tieske said we can merge with this TODO in place, because it's fixed in the other upstream PR.